be/src/formats/parquet/encoding_dict.h

```diff
@@ -113,16 +113,12 @@ class DictDecoder final : public Decoder {
     Status skip(size_t values_to_skip) override {
         //TODO(Smith) still heavy work load
         _indexes.reserve(values_to_skip);
-        _index_batch_decoder.GetBatch(&_indexes[0], values_to_skip);
+        _rle_batch_reader.GetBatch(&_indexes[0], values_to_skip);
         return Status::OK();
     }
 
     Status next_batch(size_t count, ColumnContentType content_type, Column* dst) override {
-        _indexes.reserve(count);
-        _index_batch_decoder.GetBatch(&_indexes[0], count);
-
-        FixedLengthColumn<T>* data_column = nullptr;
-
+        FixedLengthColumn<T>* data_column /* = nullptr */;
```

Comment:
variables needs initialized in CPP to prevent compiler from complaining, it is quite different from Java and C